### ISSUE TITLE
Nether wart doesn't grow, even in Nether

### ISSUE THREAD

**Issue by migmolrod** [ChosenArchitect/Bounds-1.12-Modpack#129]
First I tried to directly plant Nether wart in a Soul sand block near my house in the Overworld. I had that type of item desync as described in this other issue: https://github.com/ChosenArchitect/Bounds-1.12-Modpack/issues/126

According to that issue, we have to use planters. So I tried (in the same spot near my house, overworld) and nether wart was stuck at 0% growth no matter what. After a few searches, I found that nether wart can only be grown in the Nether dimension. I went there and put my planters. Still stuck at 0% no matter what.
I even tried to directly plant warts in plain soul sand (no planters involved) but the wart directly disappears like in the abovementioned issue.

Any ideas? Using latest version with MultiMC (using Linux, can't use Twitch app to install modpacks; interestingly enough, MC runs way smoother than in my Windows partition)

**Comment by rujuk** [comment]
I forgot to mention in #126 that I used bonemeal to grow my nether warts, also in overworld.
But i didnt notice before, they do not grow on their own in any dimension, apparently.

Althougth in the NetherEx config the Nether Wart are set to grow in vanilla speed

Tip:
There are traders in nether that sell warts or once you get your skeleton spawner you will have bones for years. I only have 1 planter with 1 nether wart. just some bonemeal and hold rightclick, no need to replant.

**Comment by migmolrod** [comment]
I'll do that as a workaround, thanks. Would love to see wart growing naturally tho.

**Comment by Ravin890** [comment]
Nether ex makes nether wart grow with ichor nearby (like farmland and water) and a gold wither bone hoe tilled soul sand

**Comment by migmolrod** [comment]
Yeah I found it out right now while reading Nether Ex documentation, trying to find info about the Urn of sorrow. It's in the same section "Special blocks". Thanks, @Ravin890 
I guess it's now time to explore and find that ichor.

**Comment by migmolrod** [comment]
New problem found: the Arctic Abyss biome from NetherEx seems to be directly disabled and AFAIK it's the only biome in which ichor can be generated.

At first I thought it was my lack of luck. Even tried to recreate my world with commands enabled and started to fly in spectator mode to search for an Arctic Abyss. Not a trace.
Then I took a look at the config files in Bounds modpack and I found the file /config/NetherEx/Biomes/nex/arctic_abyss.json where the property "enable" is set to false, which I suppose means it's effectively disabled and the world generator doesn't even know there is an Arctic Abyss.

Is this intended in terms of balance to have bone meal as the only way to grow nether wart or is it a 'misconfiguration'? @ChosenArchitect
